packages/ibm-products/src/components/StatusIndicator/StatusIndicatorStep.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg } from '../../settings';
+
+import { InlineLoading } from '@carbon/react';
+import {
+  CheckmarkOutline,
+  CircleStroke,
+  ErrorOutline,
+} from '@carbon/react/icons';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--status-indicator-step`;
+const componentName = 'StatusIndicatorStep';
+
+const defaults = {
+  status: 'inactive',
+};
+
+/**
+ * An icon/description pair that describes one step of the `StatusIndicator`.
+ */
+export let StatusIndicatorStep = React.forwardRef(
+  (
+    { className, description, errorMessage, status = defaults.status, ...rest },
+    ref
+  ) => {
+    return (
+      <li
+        {...rest}
+        className={cx(blockClass, className, `${blockClass}--${status}`)}
+        ref={ref}
+        {...getDevtoolsProps(componentName)}
+      >
+        <div className={`${blockClass}--details`}>
```

Comment:
Small detail, but to match the class name structure of the other classes in your component I think it makes sense for this class to be `${blockClass}__details`.
```suggestion
        <div className={`${blockClass}__details`}>
```

Reply:
Oops. My bad.